geonode_mapstore_client/client/js/components/FiltersForm/FilterByExtent.jsx

```diff
@@ -25,8 +25,9 @@ function ZoomTo({
     map,
     extent
 }) {
+    const once = useRef();
     useEffect(() => {
-        if (map && extent) {
+        if (map && extent && !once.current) {
             const [
                 minx, miny, maxx, maxy
             ] = extent.split(',');
```

Comment:
It's missing part of the solution proposed [here](https://github.com/GeoNode/geonode-mapstore-client/pull/187#discussion_r638617495). Was there some issue on that code?
That part is needed when you apply an extent that crosses the IDL and close then open the filter again. Please check if it's working and if so please apply it.

Reply:
okay

Reply:
Tested this and works

Reply:
thanks